di_art/lib/time/Measure.dart

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 DiArt.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import 'package:di_art/utils/Pair.dart';
+
+double measureDurationOnly(Function code){
+  int start = DateTime.now().microsecondsSinceEpoch;
+  code();
+  return (DateTime.now().microsecondsSinceEpoch - start) / 1000000.0;
```

Comment:
As far as I understood you use seconds as your target measure unit, don't you? Do you think it has enough precision?

Reply:
I've resolved the issue with compilation :)

Reply:
But I would rather stick to the more precise solution. It is always possible to come to less precise number but vise versa is not possible :)

Reply:
@creageek @s0m3chill What do you think?

Reply:
This is primarily migration from cocaine. and all values ​​are taken from there.

Reply:
This is primarily migration from koin. and all values ​​are taken from there.

Reply:
I am sure that in the future this will make sense in further migration.